common/time_win.h

```diff
@@ -0,0 +1,12 @@
+//
+// Created by aveen on 2020-08-24.
+//
+
+#ifndef YUBIHSM_SHELL_TIME_WIN_H
+#define YUBIHSM_SHELL_TIME_WIN_H
+
+#include <time.h>
+
+int gettimeofday_win(struct timeval *__restrict tv);
```

Comment:
is the __restrict keyword needed here?

Reply:
Probably not. I had a different approach at the beginning and so used the same declaration as gettimeofday(). I just didn't change the declaration enough after that. 

Removed it now